kanidm_book/src/installing_the_server.md

```diff
@@ -1,6 +1,8 @@
 # Installing the Server
 
-> **NOTE** Our preferred deployment method is in containers, the documentation assumes you're running in docker. Kanidm will run in traditional compute, and server builds are available for multiple platforms or you can build the binaries yourself.
+> **NOTE** Our preferred deployment method is in containers, the documentation assumes you're running in docker. Kanidm will run in traditional compute, and server builds are available for multiple platforms, or you can build the binaries yourself.
+
+TODO why are containers preferred?
```

Comment:
@Firstyear could answer this one 😄

Reply:
Because packaging the kanidm server for most distros is a shit show. 🙃 It's a lot easier if we say "hey use a container" then we at least have a smaller surface area for things to go wrong.